src/Browser.mad

```diff
@@ -0,0 +1,6 @@
+// !TODO: change when nullary functions land
+getLocation :: () -> String
+export getLocation = (_) => #- window.location.pathname -#
+
+decodeURI :: String -> String
+export decodeURI = (x) => #- decodeURIComponent(x) -#
```

Comment:
Have a look at that one: https://madlib-lang.github.io/madlib/#/URL. It uses `decodeURI` under the hood though, not sure of the difference, otherwise we can add it there anytime.